### PR TITLE
Remove TableLayout from Accumulo connector

### PR DESCRIPTION
**Pull request by Praveen2112**
Removes the TableLayout for Accumulo connector. (#781)

### CODE REVIEW

File: presto-accumulo/src/main/java/io/prestosql/plugin/accumulo/AccumuloMetadata.java [hunk 1]
```diff
@@ -352,6 +334,42 @@ public void renameColumn(ConnectorSession session, ConnectorTableHandle tableHan
...
+        if (oldDomain.equals(newDomain)) {
+            return Optional.empty();
+        }
+
+        handle = new AccumuloTableHandle(
+                handle.getSchema(),
+                handle.getTable(),
+                handle.getRowId(),
+                newDomain,
+                getColumnConstraints(handle.getRowId(), newDomain),
```
Comment (martint) [inline presto-accumulo/src/main/java/io/prestosql/plugin/accumulo/AccumuloMetadata.java]: Why do we need to hold on to the column constraints in the table handle? It seems like we should be able to do this directly where they are needed (i.e., in `getSplits`)

Comment (Praveen2112) [reply presto-accumulo/src/main/java/io/prestosql/plugin/accumulo/AccumuloMetadata.java]: Column constraints are also similar to `TupleDomain` which would remain same for all splits so thought of moving it to the `AccumuloTableHandle`.

Comment (martint) [reply presto-accumulo/src/main/java/io/prestosql/plugin/accumulo/AccumuloMetadata.java]: As far as I can tell, they are only used in `getSplits()` to call `client.getTabletSplits(...)`, but they are not stored in splits. So we could derive them directly in `getSplits()` from the information in the handle.

Comment (Praveen2112) [reply presto-accumulo/src/main/java/io/prestosql/plugin/accumulo/AccumuloMetadata.java]: It was passed to `AccumuloRecordSet` and `AccumuloRecordCursor` but it is not used anywhere expect in getting the split metadata. So we can maintain it as `TupleDomain` and derive it from table handle.

**Comment by Praveen2112** [comment]
@martint Have removed the column constraint from the table handle. Please let me know if there is any more changes.

**Pull request merged**